<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfazOcular</class>
 <widget class="QMainWindow" name="InterfazOcular">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfazOcular</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Btn_L">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>500</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_R">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>499</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Btn_Settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuracion</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>69</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_6">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Yaw:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="in_Y">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="in_X">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Roll:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="in_Z">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="in_Pitch">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="in_Roll">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="in_Yaw">
       <property name="maximumSize">
        <size>
         <width>69</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Box_faceVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>209</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Box_armVideo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>9</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>500</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Option_Left2">
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Left2</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Option_Left1">
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Left1</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Up">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>59</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Option_Right1">
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Right1</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Option_Right2">
       <property name="maximumSize">
        <size>
         <width>153</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Right2</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
